Public/Sdk/Public/Managed/Shared/types.dsc

```diff
@@ -78,6 +78,11 @@ export interface Assembly extends Deployment.Deployable {
      * that conflict with other dependencies.
      */
     runtimeContentToSkip?: Deployment.DeployableItem[];
+
+    /**
+     * Similiar to runtimeContent, but specific to native files.

```

Comment:
I don't see the difference.  `runtimeContent` can specify any files, native or not.  Why couldn't you continue to use `runtieContent`?